src/controllers/accounts.js

```
@@ -151,4 +193,28 @@ accountsController.postAccount = function(req, res, next) {
     res.sendStatus(200);
 };
 
+accountsController.uploadImage = function(req, res, next) {
+    var self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)